_im-plugin/ism/error-prevention/api.md

```diff
@@ -0,0 +1,155 @@
+---
+layout: default
+title: ISM error prevention API
+parent: ISM error prevention
+nav_order: 10
+---
+
+# ISM Error prevention API
+
+The ISM Error Prevention API allows you to enable ISM error prevention, as well as checking the validation status and message.
```

Comment:
Suggestion only:
"...enable ISM error prevention and check the validation status ..."